### ISSUE TITLE
Create Thunderbird 'Standard' flow

### ISSUE THREAD

**Issue by edrushka** [mozilla/donate.mozilla.org#1281]
Create a version of the 'Standard' donation flow https://donate.mozilla.org/en-US/ for Thunderbird.

Copy being finalized in #1280 

Assigning @alicoding but tagging @ScottDowne to provide support

**Comment by edrushka** [comment]
@davidascher do you want metrics for this?

**Comment by edrushka** [comment]
@alicoding or @ScottDowne can you please give me a general idea of where this is at? When we get final copy from Gerv, how long will it take to set this up (not including the styling tweaks that Ricardo will suggest)?

@davidascher what sort of metrics do we need here?

**Comment by ScottDowne** [comment]
@edrushka I initially gave an estimate of a week to get this up, after design is done.

It'll take less than that if it's solid work, but I highly doubt we'll get solid time.

Also, I won't be the one doing the actual work, as I'll be on PTO during this.

**Comment by alicoding** [comment]
I don't know about the timing yet, but I have put some thought about this
and discussed with @jbuck briefly about the solution to this Thunderbird
donation flow. I'm not sure if it will be straightforward to just create
another page on the same repo since all of our API endpoints were made
specifically for EOY campaign, so about a week is reasonable for tech work,
but I would assume we will have to do non tech stuff like creating PayPal
account or setup Stripe and ET Basket?

**Comment by edrushka** [comment]
Thanks guys. Who is able to setup the Paypal Accounts, Stripe and ET basket?  @Jbuck? That seems like a distinct chunk of work we can do soonish.

**Comment by davidascher** [comment]
I don't think we need separate accounts as we're doing the accounting based
on URLs or campaign IDs.

**Comment by edrushka** [comment]
OK thanks for clarifying - I'm still trying to get my head around what needs to happen first. 

If we're not setting up new accounts, @alicoding are there any blockers preventing you from getting started on this?

**Comment by jbuck** [comment]
We need to talk to Alex about how he currently splits out the Firebug donations for Paypal, and see if we can do the same for Stripe.

**Comment by rkent** [comment]
I'm trying to understand all of these pieces, and see what I can do to move things forward.

As I understand it, this "Standard" flow starts from a site/snippet that has content that encourages the user to give. For Thunderbird, this would be the in-product start page https://www.mozilla.org/en-US/thunderbird/release/start/ (or the localized, version-specific version) which is a bedrock-hosted page generated as a redirect from the in-product link which is currently something like https://live.mozillamessaging.com/thunderbird/start/?locale=en-US&version=38.5.0 but that we are probably going to change to the thunderbird.net domain (See bug 1247721 and bug 1229640). Is the donate campaign bedrock page in scope for this donate.mozilla.org/MoFo project? I assume that the redirects are out of scope.

The bedrock page links to a donate.mozilla.org page, with the Firefox variant being https://donate.mozilla.org  I assume that we are talking about a Thunderbird variant of this page as the "Standard" flow, and the "With Context" flow is a link that does not assume an earlier context, but to which someone arrives from simply clicking on a "donate" link.

Is this current task simply the creation of that one page, or is it the entire workflow including the thank you email, setup of email list signup, etc.

donate.mozilla.org has very little firefox-specific content on it, but that page itself is not appropriate for the Thunderbird landing page for a variety of reasons. Should I list those reasons and propose changes? I don't really know how MoFo typically organizes work on these kinds of things. If it was me, I would just copy the existing MoFo page as a prototype, but get the prototype up so that we could start changing things that need work. How would you like me to interact with you on this?

**Comment by rkent** [comment]
See my comments on #1280 where I proposed specific changes to this page.

**Comment by edrushka** [comment]
@davidascher - can you please respond to @rkent's question about a donate campaign bedrock page.

Yes, @rkent you are correct that anyone redirected from a source with context will go to the standard flow and links without context go to the "with context" flow that has the additional text.

This task, what we call the "flow", is the donate page and automatic Stripe & Paypal messages:
- a [donate form] (https://donate.mozilla.org/en-US/)
- a 'thank you'/'donation confirmed' message discussed in #1293

Any additional follow-up emails should be set up and owned by the TB team as discussed in #1289

The content questions are detailed in #1280. I think we're all on the same page there.

Now that we have some draft copy to work with, I think we can get a prototype up. Obviously some details still need to be ironed out, but I think having something on staging will help bring clarity.

**Comment by wsmwk** [comment]
This, #1280 and others make sense to me. I'll wait to see draft/staging before commenting further, unless kent needs me to check something specific before then.

**Comment by alicoding** [comment]
I just spoke to Alex about how should we do this for PayPal and Stripe and his suggestion was to tag Thunderbird with the description field that we send to PayPal and Stripe which will help us filter out to see the contribution specifically to Thunderbird only and we can use the same account for both current donate.mozilla.org

**Comment by alicoding** [comment]
I think we can cal lthis done https://donate.mofostaging.net/en-CA/thunderbird/

**Comment by davidascher** [comment]
W00t
